pandas/tests/types/test_cast.py

```diff
@@ -9,33 +9,33 @@
 import numpy as np
 
 from pandas import Timedelta, Timestamp, DatetimeIndex
-from pandas.types.cast import (_possibly_downcast_to_dtype,
-                               _possibly_convert_objects,
-                               _infer_dtype_from_scalar,
-                               _maybe_convert_string_to_object,
-                               _maybe_convert_scalar,
-                               _find_common_type)
+from pandas.types.cast import (maybe_downcast_to_dtype,
+                               maybe_convert_objects,
+                               infer_dtype_from_scalar,
+                               maybe_convert_string_to_object,
+                               maybe_convert_scalar,
+                               find_common_type)
 from pandas.types.dtypes import (CategoricalDtype,
                                  DatetimeTZDtype, PeriodDtype)
 from pandas.util import testing as tm
 
 
-class TestPossiblyDowncast(tm.TestCase):
+class TestMaybeDowncast(tm.TestCase):
```

Comment:
I wasn't quite sure whether or not I should change the name of this class. It seemed logical as it seemed to refer to the "private" method I had changed.